pkg/kudoctl/cmd/install/install.go

```diff
@@ -36,6 +42,26 @@ func CmdErrorProcessor(cmd *cobra.Command, args []string) error {
 	return nil
 }
 
+func validateInstallParameters() error {
+	if vars.Parameter != nil {
+		for _, a := range vars.Parameter {
+			// Using '=' as the delimiter. Split after the first delimiter to support using '=' in values
+			s := strings.SplitN(a, "=", 2)
+			if len(s) < 2 {
+				return fmt.Errorf("parameter not set: %+v", s)
```

Comment:
I'd include `a` (the full arg, e.g., `foo` and not `s` (the array resulting from the split) in the error.

That way if a parameter is just `foo`, the resulting error would be: `parameter not set: foo` instead of `parameter not set: [foo]`.